addons/loyalty_delivery/tests/test_loyalty_delivery.py

```diff
@@ -0,0 +1,99 @@
+
+from odoo.tests import common, Form
+from odoo.exceptions import ValidationError
+
+@common.tagged('post_install', '-at_install')
+class TestLoyaltyDeliveryCost(common.TransactionCase):
+
+    def setUp(self):
+        super(TestLoyaltyDeliveryCost, self).setUp()
+        self.SaleOrder = self.env['sale.order']
+        self.partner_1 = self.env['res.partner'].create({'name': 'My Test Customer'})
+        self.pricelist = self.env.ref('product.list0')
```

Comment:
FYI `list0` pricelist is dropped in 16.2, so you probably want to directly create a pricelist manually here to avoid a red runbot on the 16.2 forwardport